test_requirements.txt

```diff
@@ -1,5 +1,14 @@
+cheroot>=6.0.0
 DBUtils
-mysql-connector>=2.1.4
-psycopg2
-PyMySQL
 pytest>=4.6.2
+
+# DB drivers.
+# mysql
+MySQL-python>=1.2.5; python_version == '2.7'
+PyMySQL>=0.9.3
+mysql-connector-python>=8.0.19
```

Comment:
Do we really need all three of these?

Reply:
Yes.

Check file `tests/test_db.py`, unittests are triggered if the db module is available / importable.